CODEOWNERS

```diff
@@ -5,5 +5,4 @@
 # Order is important: later rules override preceding rules
 
 # These owners will be the default owners for everything in the repo unless a later match takes precedence
-* @mesosphere-eng-noctilucent
-/stable/dispatch @devx
```

Comment:
Why were the dispatch codeowners removed?

Reply:
Oh, I guess because we thought that `@mesosphere/d2iq-ksphere` would cover everyone who needs access.

Nevertheless, since @mesosphere/sig-ksphere-catalog is the owner of this repo now, we're good.